helm-charts/flink-historyserver/values.yaml

```diff
@@ -0,0 +1,3 @@
+aws_efs_file_system_id: ""
+# this helps us bump it and reapply via TF
+version: "v1"
```

Comment:
Can you help me understand why this is needed? In general, redeployment is necessary only when config values change, right? If the *contents* of the chart change, we can use the `version` argument in `Chart.yaml` to redeploy instead.

Reply:
it didn't seem to be picking up and applying changes from the `Chart.yaml`. So I just came up with this quickly so it would redeploy. But I was moving fast and probably doing something wrong. So I will give this another look

Reply:
that worked, I was wrong